tools/tests/centos7/gf-builder

```diff
@@ -3,18 +3,22 @@ FROM centos:7.9.2009
 ENV container docker
 ENV PYTHON /usr/bin/python2
 
-RUN yum install -y systemd epel-release
+#RUN yum install -y --setopt=tsflags=nodocs systemd epel-release && yum clean all
```

Comment:
Why comment it instead of removing the line ?

Reply:
Oops. Will fix.